lib/screens/settings/donations.dart

```diff
@@ -80,11 +80,14 @@ class DonationsScreen extends StatelessWidget {
                                               return MobileScanner(
                                                   allowDuplicates: false,
                                                   onDetect: (barcode, args) {
-                                                    DonationsAdapter.instance
-                                                        .setRealtimeCashAddress(
-                                                            address: barcode
-                                                                .rawValue!
-                                                                .toLowerCase());
+                                                    if (barcode.rawValue !=
+                                                        null) {
+                                                      DonationsAdapter.instance
+                                                          .setRealtimeCashAddress(
+                                                              address: barcode
+                                                                  .rawValue!
+                                                                  .toLowerCase());
+                                                    }
```

Comment:
```suggestion
                                                    final value = barcode.rawValue;
                                                    if (value != null) {
                                                      DonationsAdapter.instance
                                                          .setRealtimeCashAddress(
                                                              address: value.toLowerCase());
                                                    }
```

We can avoid the non-null assertion. Might need a dart fmt.